api.py

```diff
@@ -111,20 +111,21 @@ async def submit_agent(submission: SubmissionCode, current_user: dict = Depends(
     if not is_agent_safe(submission.code):
         return ErrorResponseModel(status="error", message="Agent code is not safe.")
     
+
     team = database.get_team(session, team_name)
     if not team.league:
         return ErrorResponseModel(status="error", message="Team is not assigned to a league.")
-    
+    ensure_test_league_folder(team.league.game)
     results = run_agent_simulation(submission.code, team.league.game, team_name)
     if not results:
         return ErrorResponseModel(status="error", message="Agent simulation failed.")
     
     try:
         print("team found", team.name, team.league.name, team.league.folder)
         if team.league.folder:
-            league_folder = team.league.folder
+            league_folder = os.path.normpath(team.league.folder).lstrip('\\')
```

Comment:
Why backslashes?

Reply:
league_folder variable is this
\league\week1\
Which breaks the path code when this is used

```python
file_path = os.path.join(ROOT_DIR, "games", team.league.game, league_folder, f"{team_name}.py")
```

Instead of the file_path being the correct one it reverts to C:\league\week1\test1.py

Adding 
```python
os.path.normpath(team.league.folder).lstrip('\\')
```
removes the issue and the file path is correctly used.

Will find a universal solution for windows and linux in terms of assigning slashes.

Reply:
We should do a linux only solution. It is too hard to support both, so no backslashes